app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -34,6 +35,10 @@ keywords:
 
 Postgraduate teaching bursaries and scholarships are only available for the subjects listed below. You cannot receive both a teaching bursary and a scholarship.
 
+You cannot receive a scholarship or bursary if your teacher training course is salaried.
```

Comment:
Minor suggestion - would it be better to switch round these two sentences:

> You cannot receive a scholarship or bursary if your teacher training course is salaried.
> 
> You must be eligible for student finance to receive a bursary or scholarship (unless you’re training to teach physics or languages).

The second one is more relevant for more people (and a bit more positive!)